test/module_6/protractorTask.spec.js

```diff
@@ -0,0 +1,55 @@
+const { browser, element, $ } = require("protractor");
+
+describe("Protractor Demo App", () => {
+    const firstNumber = element(by.model("first"));
+    const secondNumber = element(by.model("second"));
+    const goButton = element(by.id("gobutton"));
+    const operator = element(by.model("operator"));
+    const history = element.all(by.css("tr.ng-scope"));
+
+    browser.get("http://juliemr.github.io/protractor-demo/");
```

Comment:
for future: I would recommend using async/await or promise chaining for async functions. I wouldn't hope on control flow.

Reply:
ok)